server/bin/pbench-audit-server.sh

```diff
@@ -727,9 +727,11 @@ if [[ $? -ne 0 ]]; then
 fi
 eTS=$(timestamp)
 if [[ -s ${archive_report} ]]; then
-    printf "\nstart-${sTS}: archive hierarchy: $ARCHIVE\n" | tee -a ${report}
+    log_info "" "${report}"
+    log_info "start-${sTS}: archive hierarchy: $ARCHIVE" "${report}"
```

Comment:
Instead of calling `log_info` twice (once with an empty arg), can't you pass the string with `\n` to `log_info`? I really dislike all the multiple calls here and below.

Reply:
Try changing the %s in the printf format specification in log_info to %b and see if that causes problems. %b is supposed to expand backslash escape sequences. Read the bash manpage, the section on printf, for details.